### ISSUE TITLE
Laundry Schedule Mishandles DST

### ISSUE THREAD

**Issue by tymorrow** [deltasig/sphinx#2]
Everything appears to be fine if you sign up for a slot before the DST switch occurs, regardless of whether or not the slot you chose occurs before or after the DST switch.  However, when the DST switch actually does occur, all of the signups that are for slots after the DST switch are not improperly converted from UTC to CST.  This is because they were recorded being an hour off since the DST switch had not yet occurred.

Basically, the logic for generating all of the slots does not detect and adjust properly for times occurring after a DST switch.
